charmcraft/application/commands/store.py

```diff
@@ -2374,3 +2374,73 @@ def _get_architectures_from_bases(
         for architecture in base.architectures:
             architectures.add(architecture)
     return sorted(architectures)
+
+
+class CreateTrack(CharmcraftCommand):
```

Comment:
What about adding any tests for this Command and/or new Service API :smile:  ?

Reply:
The initial version didn't really have enough logic in it to write tests against, but I realised I probably shouldn't be pushing *all* tracks back, so I've filtered it and that was sufficient excuse to write tests.

Reply:
Oh, and I just realised I didn't include my spread test changes originally. Oops!